src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (Storage.fruits.containsKey(transaction.getFruit())

```

Comment:
The condition checks if the fruit is in the storage but does not handle the case when the fruit is not present. This could lead to errors or unexpected behavior.

Reply:
ok, done